### ISSUE TITLE
Security policy

### ISSUE THREAD

**Issue by setharnold** [canonical/starbase#82]
Hello, please consider setting up a security policy for the repository, and populating a SECURITY.md file. These policies and files are useful for security researchers and the general public who would like to report issues in a way to minimize risk.

See https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository for more information.

Thanks

**Comment by lengau** [comment]
I agree in principle, but to be perfectly honest I'm not sure what our security policy would be

Hopefully we can get traction on https://github.com/canonical/.github/issues/6, which would give us some guidance for how individual projects or teams should tackle it.

**Comment by sergiusens** [comment]
Solved with #356
